app/Models/Foundation/Summit/Sponsor.php

```diff
@@ -809,5 +817,48 @@ public function setShowLogoInEventPage(bool $show_logo_in_event_page): void
         $this->show_logo_in_event_page = $show_logo_in_event_page;
     }
 
+    /**
+     * @return SummitSponsorExtraQuestionType[]
+     */
+    public function getExtraQuestions()
+    {
+        return $this->extra_questions;
+    }
+
+    /**
+     * @param int $extra_question_id
+     * @return SponsorSocialNetwork|null
+     */
+    public function getExtraQuestionById(int $extra_question_id): ?SummitSponsorExtraQuestionType
+    {
+        $criteria = Criteria::create();
+        $criteria->where(Criteria::expr()->eq('id', intval($extra_question_id)));
+        $extra_questions = $this->extra_questions->matching($criteria)->first();
+        return $extra_questions === false ? null : $extra_questions;
+    }
+
+    /**
+     * @param ExtraQuestionType $extra_question
+     */
+    public function addExtraQuestion(ExtraQuestionType $extra_question): void
+    {
+        if ($this->extra_questions->contains($extra_question)) return;
+        $this->extra_questions->add($extra_question);
+        $extra_question->setSponsor($this);
```

Comment:
you are omitting the order
please check summit implementation for order question
here
and the use of trait OrderableChilds
https://github.com/OpenStackweb/summit-api/blob/a02088453cff6fb953998a34f15f66a9d107ce17/app/Models/Foundation/Summit/Summit.php#L4410